src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "fusedgemm.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "ngraph/builder/reshape.hpp"
+#include "ngraph/op/add.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/matmul.hpp"
+#include "ngraph/op/multiply.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector fusedgemm(const Node& node) {
+    OutputVector inputs{node.get_ng_inputs()};
+    Output<ngraph::Node> input_a = inputs.at(0);
+    Output<ngraph::Node> input_b = inputs.at(1);
+    Output<ngraph::Node> input_c;
+
+    if (inputs.size() == 3) {
```

Comment:
Please add `#include "onnx_import/core/null_node.hpp"` and use the `op::is_null` helper to check the optional
```suggestion
    if (inputs.size() == 3 && !ngraph::op::is_null(inputs[2])) {
```

Reply:
ok, i will follow your suggestion.